packages/api-clients/open-api/authorizationApi.yml

```
@@ -1154,7 +1154,6 @@ paths:
         - name: producerId
           in: query
           description: ID of producer that MUST be related to the keychain
-          required: true
```

Why this change? For clients, the `consumerId` is mandatory, and we did not change that during the visibility refactoring. I would do the same (but for producerId) here.
See https://vscode.dev/github/pagopa/interop-be-monorepo/blob/release/2.5.0/packages/api-clients/open-api/authorizationApi.yml#L218-L219

```suggestion
          required: true
```